src/components/CookieConsent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import 'vanilla-cookieconsent/dist/cookieconsent.css';
+import * as CookieConsent from 'vanilla-cookieconsent';
+import LeadinfoScript from '../components/layout/leadinfo-script';
+
+const CookieConsentComponentV3 = () => {
+  const [analyticsEnabled, setAnalyticsEnabled] = useState(false);
+
+  useEffect(() => {
+    CookieConsent.run({
+      categories: {
+        necessary: {
+          enabled: true,
+          readOnly: true,
+        },
+        analytics: {
+          enabled: false,
+        },
+      },
+      language: {
+        default: 'en',
+        translations: {
+          en: {
+            consentModal: {
+              title: 'We use cookies',
+              description:
+                'This website uses cookies to ensure the best user experience.',
+              acceptAllBtn: 'Accept all',
+              acceptNecessaryBtn: 'Reject all',
+              showPreferencesBtn: 'Manage preferences',
+            },
+            preferencesModal: {
+              title: 'Manage Cookie Preferences',
+              acceptAllBtn: 'Accept all',
+              acceptNecessaryBtn: 'Accept necessary only',
+              savePreferencesBtn: 'Save preferences',
+              closeIconLabel: 'Close',
+              sections: [
+                {
+                  title: 'Strictly Necessary Cookies',
+                  description:
+                    'These cookies are essential for website functionality and cannot be disabled.',
+                  linkedCategory: 'necessary',
+                },
+                {
+                  title: 'Analytics Cookies',
+                  description:
+                    'We use analytics cookies to analyze website usage and improve our services.',
+                  linkedCategory: 'analytics',
+                },
+              ],
+            },
```

Comment:
1. Is there no default language shipped?
2. Put them into a separate en file, I think that's manageable and makes it less noisy to work with the config.